cmd/manager/main.go

```diff
@@ -240,6 +240,10 @@ func parseConfiguration() Config {
 			"Enabling this will ensure there is only one active controller manager.")
 	flag.StringVar(&config.LogLevel, "log-level", "info", "Log level. Available values: debug | info | warn | error | dpanic | panic | fatal")
 	flag.StringVar(&config.LogEncoder, "log-encoder", "json", "Log encoder. Available values: json | console")
+	flag.BoolVar(&config.ObjectDeletionProtection, objectDeletionProtectionFlag, true, "Defines the operator will not delete Atlas resource "+
+		"when a Custom Resource is deleted")
+	flag.BoolVar(&config.SubObjectDeletionProtection, subobjectDeletionProtectionFlag, true, "Defines that the operator will not overwrite "+
+		"(and consequently delete) subresources that were not previously created by the operator")
```

Comment:
We must remember to remove these CLI flags again, and the associated variables. It is due to the rebase. Same happened to my PR.

I will remind in Slack as well.